jumble/src/hooks/use-charm.ts

```diff
@@ -18,9 +18,12 @@ const loadCharmData = async (
   let iframeRecipe = null;
 
   if (charm) {
-    await charmManager.syncRecipe(charm);
-    const ir = getIframeRecipe(charm);
-    iframeRecipe = ir?.iframe ?? null;
+    try {
+      const ir = getIframeRecipe(charm);
+      iframeRecipe = ir?.iframe ?? null;
+    } catch (e) {
+      console.info(e);
```

Comment:
I think `console.warn('Failed to find iframe recipe' e);` would be more appropriate

Reply:
I was debating this - as this most likely means it isn't an iframe charm. I think I should have a way to check "is iframe" first - and then if getting the recipe fails then explode

Reply:
leaving this for now - happy to change in next round